src/migrations/20231030150724-sim_score_cache.js

```diff
@@ -0,0 +1,37 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.createTable(
+        'SimScoreCaches',
+        {
+          id: {
+            type: Sequelize.INTEGER,
+            allowNull: false,
+            primaryKey: true,
+            autoIncrement: true,
+          },
+          recipient_id: { allowNull: false, type: Sequelize.INTEGER },
+          goal1: { allowNull: false, type: Sequelize.INTEGER },
```

Comment:
should these be noted as references in the migration? 

```js
{
   type: Sequelize.INTEGER,
   allowNull: true,
   default: null,
   references: {
      model: {
      tableName: 'Goals',
   },
   key: 'id',
}
 ```

Reply:
yup! thank you

Reply:
539b79ae0c62125b04f9073e935387d0f03d7224